.github/workflows/java.yaml

```diff
@@ -14,23 +14,27 @@ jobs:
   build:
     name: Build
     runs-on: ${{ matrix.os }}
+    container: ${{ matrix.container }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        include:
+          - os: ubuntu-latest
+            # Use Fedora container to get new enough Meson
```

Comment:
I just use `pipx install meson` to get a uptodate version, right now I have 0.64.1 on Ubuntu 20.04.1 at home. I also install a current version of ninja that way, not sure if the java build depends on that or not.
Now if you don't want to deal with setting up pipx and adding `~/.local/bin` to the path and all that, just pip install it system-wide and be done.